CODING_GUIDELINES.md

```diff
@@ -207,7 +207,36 @@ class MyClass
 }
 ```
 
-### 8. Misc.###
+### 8. Header inclusion order.###
+The headers should be placed in the following order:
+ 1. Module header (in .cpp)
+ 2. System/Qt/Boost etc. headers (splitted in subcategories if you have many).
```

Comment:
For me it is redundant information. I know, from any of the modules qBittorrent depends overall, and that's enough for me.